osu.Game.Rulesets.Catch/Edit/CatchSelectionHandler.cs

```diff
@@ -26,13 +29,19 @@ public override bool HandleMovement(MoveSelectionEvent<HitObject> moveEvent)
             Vector2 targetPosition = HitObjectContainer.ToLocalSpace(blueprint.ScreenSpaceSelectionPoint + moveEvent.ScreenSpaceDelta);
             float deltaX = targetPosition.X - originalPosition.X;
 
+            foreach (float x in EditorBeatmap.SelectedHitObjects.SelectMany(getOriginalPositions))
+                deltaX = Math.Clamp(deltaX, -x, CatchPlayfield.WIDTH - x);
+
+            if (deltaX == 0)
+            {
+                // Returns true: even there is no positional change, there may be a time change.
+                return true;
```

Comment:
Not much point in commenting that the method returns true before returning true. Just the explanation suffices.

Reply:
The intention was it is a comment about the returning value, not that it is returning early.

Reply:
Probably fine as
```suggestion
                // even there is no positional change, there may be a time change.
                return true;
```